### PR TITLE
fix: navbar exception

### PR DESCRIPTION
**Pull request by dricazenck**
## Description

Check if the object exists before assigning a class to avoid undefined exceptions.

## Change Type
- [x] Bug Fix

## Related Issue
Fix issue #160 

## Screenshots
Before: image

After: 
image

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: assets/js/navbar.js [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
-      // Format the text as provided in data, e.g code of conduct => "Code of conduct"
       var searchLinkText = stringPathSegment[0].toUpperCase() + stringPathSegment.slice(1, stringPathSegment.length);
       activeLink = jQuery("a.nav-link:contains('" + searchLinkText + "')");
       activeSubLink = jQuery("a.nav-link[href='" + firstPathSegment + "']")
     }
 
-    activeLink.addClass("active-nav");
-    activeSubLink.addClass("active-nav");
+    activeLink.addClass(NAVBAR_CLASS_ACTIVE);
+    if (activeSubLink) {
```
Comment (khairahscorner) [inline assets/js/navbar.js:22]: LGTM✨

File: assets/js/navbar.js [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
-  var init = function () {
-    // Get the first element of the pathname and transform
-    var pathSegments = jQuery(location).attr('pathname').split("/");
-    //transform to get links with text corresponding to url
-    // e.g /london/code-of-conduct => "code of conduct"
+  const NAVBAR_CLASS_ACTIVE = 'active-nav'
+
+  var selectActiveMenu = function () {
+    var pathSegments = jQuery(location).attr('pathname').split('/');
+
```
Comment (khairahscorner) [inline assets/js/navbar.js:6]: P.S I like that this was refactored with self-explaining variable names and the active class was also defined upfront. minimises possible errors when referencing it in later parts of the code✅

**Review by khairahscorner — APPROVED** [review]

**Review by nora-weisser — APPROVED** [review]
LGTM! 👍

File: assets/js/navbar.js (hunk 2 above)
Comment (nora-weisser) [reply assets/js/navbar.js:6]: Agree 👍

**Pull request merged**